src/models/media-file.js

```diff
@@ -146,42 +143,6 @@ export default class MediaFile extends Model {
           type: DataTypes.TEXT,
           allowNull: false,
         },
-        citation_article_title: {
-          type: DataTypes.TEXT,
-          allowNull: false,
-        },
```

Comment:
Just to double check, are we using media-files-x-bibliographic-reference.js to replace those values? Do we have any pipeline to dump those values?